core/src/test/java/org/apache/iceberg/TestCatalogUtil.java

```diff
@@ -87,6 +87,32 @@ public void loadCustomCatalog_NotImplementCatalog() {
         () -> CatalogUtil.loadCatalog(TestCatalogNoInterface.class.getName(), name, options, hadoopConf));
   }
 
+  @Test
+  public void loadCustomCatalog_ConstructorErrorCatalog() {
+    Map<String, String> options = new HashMap<>();
+    options.put("key", "val");
+    Configuration hadoopConf = new Configuration();
+    String name = "custom";
+
+    String impl = TestCatalogErrorConstructor.class.getName();
+    AssertHelpers.assertThrows("must be able to initialize catalog",
```

Comment:
Nit: "must not be able to"?

Reply:
all the other tests use this format 'must be able to', 'must have' etc.